### PR TITLE
Readme: add introduction to Chain Core

### PR DESCRIPTION
**Pull request by danrobinson**
This introduction is taken verbatim from the introduction to the Chain Core docs (except it is collapsed into one paragraph).

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Chain Core Developer Edition
 
+Chain Core is software designed to operate and participate in permissioned blockchain networks. Each network maintains a cryptographically-secured multi-asset shared ledger. Using this ledger, participants can issue digital assets directly to custodians, who can then transfer them to each other in real time with no transactional intermediary. Each Chain Core holds a copy of the ledger and independently validates each update. A federation of block signers ensures global consistency of the ledger. Digital assets share a common, interoperable format and can represent any units of value that are guaranteed by a trusted issuer — such as currencies, bonds, securities, IOUs, or loyalty points.
```
Comment (devongundry) [inline Readme.md]: not that we have assets, accounts, and blocks, would be good to add "transactions" as well @bobg

Comment (devongundry) [inline Readme.md]: maybe:

Once issued, assets are held in accounts and may be transferred among them in real time with no transactional intermediary. Each issuance or transfer is recorded in the ledger with a cryptographically signed transaction...

Comment (danrobinson) [inline Readme.md]: To get a little nitpicky—I don't think a blockchain is a ledger. - blockchain is a cryptographically secured transaction log. It allows participants to maintain a ledger. I'll try tweaking accordingly.

Comment (devongundry) [inline Readme.md]: 👍

**Comment by ryandotsmith** [comment]
@danrobinson Is this still relevant in light of #118?

**Comment by bobg** [comment]
I have some further changes awaiting approval in https://github.com/chain/chain/pull/111

**Comment by danrobinson** [comment]
I may still make some of these changes to the docs intro but I can open that separately; thanks for pointing out. Closing this one.

**Pull request closed without merging**